Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -477,6 +477,125 @@ void TextureCacheBase::BindTextures()
   }
 }
 
+class ArbitraryMipmapDetector
+{
+private:
+  using PixelRGBA = std::array<u8, 4>;
+  using PixelRGBAf = std::array<float, 4>;
+
+  static float SRGBToLinear(u8 sRGBbyte) {
```

Comment:
`srgb_byte` variables aren't capitalized. Private functions should also be below the public API wherever possible (as far as I can tell, only the typedefs are needed here.